tasks/scenario/task.ts

```diff
@@ -26,14 +25,12 @@ function getBasesFromTaskArgs(givenBases: string | undefined, env: HardhatRuntim
 task('scenario', 'Runs scenario tests')
   .addOptionalParam('bases', 'Bases to run on [defaults to all]')
   .addFlag('spider', 'run spider persistently before scenarios')
-  .addOptionalParam('stall', 'milliseconds to wait until we fail for stalling', 240_000, types.int)
-  .addOptionalParam('workers', 'count of workers', 1, types.int) // TODO: optimize parallelized workers better (1 per base?)
```

Comment:
Curious why we are getting rid of workers? Is it mainly for simplicity?

Reply:
Yeah the first step was to remove the workers to simplify to be able to do the refactor (which I don't think anyone is using now as it defaults without them since a few months ago)